### DISCUSSION TITLE
"Please edit configuration/connection/logging settings" message running suite tests

### DISCUSSION THREAD

**Discussion by gordthompson** [sqlalchemy/alembic#1159] [Usage Questions] [answered]
When running the Alembic suite tests on a third-party dialect (CockroachDB), many tests emit the message

> Please edit configuration/connection/logging settings in '/home/gord/git/sqlalchemy-cockroachdb/scratch/test_alembic.ini' before proceeding.

```
Creating directory /home/gord/git/sqlalchemy-cockroachdb/scratch/scripts ...  done
Creating directory /home/gord/git/sqlalchemy-cockroachdb/scratch/scripts/versions ...  done
Generating /home/gord/git/sqlalchemy-cockroachdb/scratch/test_alembic.ini ...  done
Generating /home/gord/git/sqlalchemy-cockroachdb/scratch/scripts/script.py.mako ...  done
Generating /home/gord/git/sqlalchemy-cockroachdb/scratch/scripts/README ...  done
Generating /home/gord/git/sqlalchemy-cockroachdb/scratch/scripts/env.py ...  done
Please edit configuration/connection/logging settings in '/home/gord/git/sqlalchemy-cockroachdb/scratch/test_alembic.ini' before proceeding.
```

The tests complete and (seem to) pass, so is this just a spurious message that can be safely ignored?

**Comment by zzzeek** [comment, marked as answer]
this is the standard out for the command that runs for "alembic init" that generates an environment.  the "please edit" is to tell the user that "alembic init" just ran and they will want to add their database URL to alembic.ini.
